final_task/rss_reader/requirements.txt

```diff
@@ -0,0 +1,5 @@
+beautifulsoup4==4.8.1
+feedparser==5.2.1
+Jinja2==2.10.3
+twine==2.0.0
```

Comment:
It seems that you are not using his lib anywhere. It is better to cleanup this file